tests/cases/properties/changes_alias.slint

```diff
@@ -0,0 +1,93 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+
+export global State {
+    // Issue #7784
+    in property <int> tab-names;
```

Comment:
That was a confusing property name - I understand that this is just a reduction though.